.gitignore

```diff
@@ -52,3 +52,6 @@ nbproject/
 # Compilation database, as may be generated by tools like Bear
 .cache/
 compile_commands.json
+.gigaide/
+.idea/
+guacamole-server.iml
```

Comment:
As with the changes in the apache/guacamole-client project, I do not believe these additions to the .gitignore file should be part of this pull request.